.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
```

Comment:
Since we are all here on Github how about evaluating GitHub actions instead of Circle CI?
you might be easier to get started than Circle CI.

https://github.com/uswat/ansible-nginx/actions/new